pkg/types/openstack/platform.go

```diff
@@ -29,6 +29,12 @@ type Platform struct {
 	// Existing Floating IP to associate with the OpenStack load balancer.
 	LbFloatingIP string `json:"lbFloatingIP"`
 
+	// ExternalDNS
+	// IP addresses of dns servers that will be added to the nodes
+	// subnet managed by the installer
```

Comment:
hmm, is `node` k8s `Nodes` ?

Also installer doesn't manage resources, so i'm maybe missing something here.

Reply:
no, its an openstack subnet created by the installer. Its a bit misworded though